### ISSUE TITLE
How to install on RPi running HAOS?

### ISSUE THREAD

**Issue by FlyMyPG** [magico13/ha-emporia-vue#305]
I just did a fresh Home Assistant 13.1 install on a Raspberry Pi 3b+ using a released image. This install seems to lack HACS, and HAOS seems to put everything in in Docker containers.

The instructions on this page make no sense (to me) for this recommended platform.

Help?

**Comment by FlyMyPG** [comment]
I have added the Github integration, but it did nothing for me.

**Comment by magico13** [comment]
Please follow the HACS installation guide first, and follow the instructions for OS/Supervised. https://hacs.xyz/docs/use

Once you've done that, add the repository to HACS. Then reboot the pi and close your web browser. Once the Pi comes back up, open your browser again and go into the normal Home Assistant Integrations page where you'd set up any of the official integrations, and search for Emporia. You should see it in that list, and once you add it you can configure your email and password.

**Comment by FlyMyPG** [comment]
It was all HACS confusion, which seems to be very common among beginners in general, and particularly among RasPi HAOS folks. Fortunately, I saw the new HACS release literally **_minutes_** after opening this issue, got the new Get-HACS, then installed HACS. Seems a bit awkward and hack-ish to do, but at least it worked.

Still, I did need your reply to clarify the final steps. In particular, the number steps and restarts/reboots needed to get from no HACS to your Emporia Vue plugin running was confusing. Does it need to be this convoluted? Then again, at least it worked in the end.

Part of the confusion is HACS itself, and I'm beginning to understand the resistance many have had to more closely integrating it with HA, much less make it part of HAOS. While I appreciate the problems surrounding finding and installing third-party repos, I'm not sure HACS is the best, or even a very good solution. I'm not sure what would be better, though perhaps something like an "Install to Home Assistant" button directly on GitHub (or via an HA GitHub plugin) could bypass the issues with HACS, much like VScode does.
